src/test_edgepi/unit_tests/test_eeprom/test_edgepi_eeprom_data.py

```diff
@@ -0,0 +1,69 @@
+'''unit test for edgepi eeprom data'''
+# pylint: disable=C0413
+# pylint: disable=no-member
+# pylint: disable=no-name-in-module
+# https://github.com/protocolbuffers/protobuf/issues/10372
+from unittest import mock
+import os
+PATH = os.path.dirname(os.path.abspath(__file__))
+import sys
+sys.modules['periphery'] = mock.MagicMock()
+
+from edgepi.eeprom.edgepi_eeprom_data import EepromDataClass
+from edgepi.eeprom.proto_files import edgepi_module_pb2
+
+def read_binfile():
+    """Read the dummy serializedFile and return byte string"""
+    with open(PATH+"/edgepi_default_bin","rb") as fd:
+        b_string = fd.read()
+    return b_string
+
+def test_init_data_class():
+    edgepi_eeprom_data = EepromDataClass()
+    edgepi_eeprom_pb = edgepi_module_pb2.EepromData()
+    assert edgepi_eeprom_pb.ByteSize() == 0
+    assert edgepi_eeprom_data.dac_calib_params is None
+    assert edgepi_eeprom_data.adc_calib_params is None
+    assert edgepi_eeprom_data.rtd_calib_params is None
+    assert edgepi_eeprom_data.tc_calib_params is None
+    assert edgepi_eeprom_data.config_key is None
+    assert edgepi_eeprom_data.data_key is None
+    assert edgepi_eeprom_data.serial is None
+    assert edgepi_eeprom_data.model is None
+    assert edgepi_eeprom_data.client_id_config is None
+    assert edgepi_eeprom_data.client_id_data is None
+    assert edgepi_eeprom_data.thing_id is None
+    assert edgepi_eeprom_data.cm_part_number is None
+    assert edgepi_eeprom_data.tb_part_number is None
+    assert edgepi_eeprom_data.cm4_part_number is None
```

Comment:
if you are expecting all of the attributes in the dataclass obj to be `None`, you can iterate with `__annotations__` instead of checking each one manually:

```
for attr in edgepi_eeprom_data.__annotations__:
    assert getattr(edgepi_eeprom_data, attr) is None
```